2-mandatory/8-journey-planner.js

```diff
@@ -19,8 +19,14 @@
   What's better about each approach?
 */
 
-function journeyPlanner(locations, transportMode) {}
-
+function journeyPlanner(locations, transportMode) {
+  let loc = [];
+  for (const location in locations) {
+   for (const value of locations[location]){
+    if (value === transportMode) loc.push(location);
+   }
```

Comment:
Code here looks more or less good but calling this function currently results in undefined. Can you see why?